foursight_core/app_utils.py

```diff
@@ -1807,6 +1806,38 @@ def run_check_runner(self, runner_input, propogate=True):
             WaitTimeSeconds=10
         )
         message = response.get('Messages', [{}])[0]
+
```

Comment:
I don't understand why the following is showing as changed. These changes were made in a previous branch/PR (check-dependencies-fix, PR-37) which was approved and merge to master, and this PR is based on that.